plugins/jobs/src/js/pages/JobDetailPage.js

```diff
@@ -45,6 +50,27 @@ class JobDetailPage extends mixin(TabsMixin) {
     return <JobRunHistoryTable job={job} />;
   }
 
+  renderBreadcrumbStates(item) {
```

Comment:
🎨 I would usually prefer to create a `JobDetailPage.Breadcrumbs` "pure subcomponent" on the top namespace. I find it a better pattern so the components are detached and you could in the future refactor them together. Also i prefere that because you don't have to make the this binding and etc.

```js
JobDetailPage.Breadcrumbs = ({item}) => {
   return ...
}

class JobDetailPage extends mixin(TabsMixin) {
....
```